requirements.txt

```diff
@@ -28,7 +27,7 @@ pywin32 ; platform_system=="Windows"
 twine
 wheel
 #py2exe; platform_system=="Windows"
-pyinstaller==5.4; platform_system=="Windows"
+pyinstaller; platform_system=="Windows"
```

Comment:
I strongly suggest to fix the version here. I had a couple of problem from time to time with pyinstaller. They keep changing the bootloader and the behavior is not the same. e.g.: the latest version was not resolving call to symbolic link. So creating a symlink of rdiff-backup in /usr/bin was not working.

Reply:
Yeah, on the other hand, it led to having a shit old version of it, being a few versions behind, so not optimal.

Reply:
After thinking about it, I prefer to cope with small incompatibilities on a regular basis, than a bunch of incompatibilities all at once, not knowing which version of PyInstaller introduced it, so I stick to my decision to remove the fixed version.